doc/tips.md

```diff
@@ -7,6 +8,7 @@
 ```
 
 (brillouinzone_sum)=
+
```

Comment:
Here a blank line is inserted. But this is less human readable and the current sphinx can manage doc source code without the blank lines. So I prefer not to insert blank lines.